packages/@aws-cdk/aws-codebuild/test/integ.github-webhook-batch.ts

```diff
@@ -0,0 +1,29 @@
+import * as cdk from '@aws-cdk/core';
+import * as codebuild from '../lib';
+
+class TestStack extends cdk.Stack {
```

Comment:
Thanks for adding these! But I think this one integ test in the CodeBuild module is enough. I would get rid of `integ.batch.lit.ts ` (and its expected file).